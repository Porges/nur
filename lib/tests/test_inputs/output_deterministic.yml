version: 1.0

default:
  after: [t1, t2, t3, t4, t5]

t5:
  run: sleep 1
t4:
  run: sleep 4
t3:
  run: sleep 3
t2:
  run: sleep 2
t1:
  run: sleep 1
