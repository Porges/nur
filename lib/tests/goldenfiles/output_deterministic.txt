stdout: |2
       t5╭╴ Started task ‘t5’
         ╰╴ Task ‘t5’ completed
       t4╭╴ Started task ‘t4’
         ╰╴ Task ‘t4’ completed
       t3╭╴ Started task ‘t3’
         ╰╴ Task ‘t3’ completed
       t2╭╴ Started task ‘t2’
         ╰╴ Task ‘t2’ completed
       t1╭╴ Started task ‘t1’
         ╰╴ Task ‘t1’ completed
  default╭╴ Started task ‘default’
         ╰╴ Task ‘default’ completed
